admin/src/App.jsx

```diff
@@ -44,9 +49,11 @@ export default function App() {
 
 				const isApiObject = generalData?.filter( ( dataset ) => dataset.id === 'api' )[ 0 ];
 				const hasApiKey = isApiObject?.options[ 'urlslab-api-key' ].value;
-
 				if ( ! hasApiKey ) {
+					setActiveOnboarding( true );
 					update( 'apiKeySet', ( ) => false );
+				} else {
```

Comment:
Would avoid else statement and rather use it non wrapped with return false in if condition